app.yaml

```diff
@@ -9,4 +12,4 @@ runtime_config:
 handlers:
 - url: /.*
   script: app.app
-# [END handlers]
\ No newline at end of file
+# [END handlers]
```

Comment:
what is this

Reply:
does not change anything. Don't know why it is being showed as a change. There is a newline at the end of the file now that was not there before